client/src/plugins/camunda-plugin/deployment-tool/DeploymentConfigModal.js

```diff
@@ -208,54 +184,40 @@ export default class DeploymentConfigModal extends React.PureComponent {
               </Modal.Title>
 
               <Modal.Body>
-                <p className="intro">
-                  { intro || 'Specify deployment details and deploy this diagram to Camunda.' }
-                </p>
-                <fieldset>
-                  <legend>
-                    Deployment Details
-                    <button
-                      type="button"
-                      className="toggle-details"
-                      onClick={ this.toggleDetails(form) }
-                      title="Toggle Advanced Details"
-                      disabled={ form.values.deployment.tenantId }
-                    >
-                      {
-                        (deploymentDetailsShown)
-                          ? <CollapseIcon className="icon" />
-                          : <ExpandIcon className="icon" />
-                      }
-                    </button>
-                  </legend>
-
+                {
+                  intro && (
+                    <p className="intro">
+                      { intro }
+                    </p>
+                  )
+                }
+                <fieldset className='first-fieldset'>
```

Comment:
Let's not use `first-child` / `last-child` marker classes.

In CSS there is better semantic support for that, either via `:first-child` or via `.intro + fieldset` selectors.

Reply:
Even better, what you'd generally do is define the distance between following fieldsets:

```css
fieldset + fieldset {
   margin-bottom: 30px;
}
```

Reply:
I squashed the first-child/last-child solution into e756492fffab26f12a9e8d892cf8a3df9aec30b9